### DISCUSSION TITLE
NCIDS Documentation Site - Can content light pages be combined?

### DISCUSSION THREAD

**Discussion by andyvanavery31** [NCIOCPL/ncids#1418] [NCIDS Launch]
The "Height and Width" and "Margin and Padding" pages are very light on content. Is there a benefit to keeping them separated or would we be able to combine them to reduce the number of pages that do not significantly contribute to the user experience?

**Comment by sarinapadilla** [comment]
We're updating these pages in #1503 and #1504.
